src/DocumenterCitations.jl

```diff
@@ -18,6 +19,9 @@ using Dates: Dates, @dateformat_str
 
 export CitationBibliography
 
+DOCUMENTER_VERSION =
```

Comment:
Instead of depending on Pkg you can use https://github.com/JuliaDocs/Documenter.jl/blob/28c5186f3c23c430d2846c199e14845ccef298ae/src/Documenter.jl#L31 directly. (Note that `Pkg.dependencies` will only be correct if Documenter is in the top level project as well)

Reply:
I'll definitely use the field in `Documenter`.

But I'm not sure that I understand what you mean: Doesn't `Pkg.dependencies` give me the version of `Documenter` in the environment where I'm currently building the documentation? I'm pretty sure that if I have a project with a main `Project.toml` and a `docs/Project.toml` and I'm building the documentation by including `docs/make.jl` in a Julia process `julia --project=docs`, the `Documenter` package only has to be a dependency in `docs/Project.toml`, not the main `Project.toml`, no?

What's the scenario where `Pkg.dependencies` would go wrong? I've used this method of getting the version of some dependency in other projects, so if I have some misconception here, I'd like to resolve it, if possible ;-)

Reply:
Yea for standard setups thats probably the case, but if someone has documenter in the default env for example:
```
julia> Base.load_path()
3-element Vector{String}:
 "/home/fredrik/dev/TestPackage/Project.toml"
 "/home/fredrik/.julia/environments/v1.9/Project.toml"
 "/opt/julia/julia-1.9.4/share/julia/stdlib/v1.9"

julia> using BenchmarkTools # loaded from second entry above

julia> using Pkg

julia> Pkg.dependencies()[Base.UUID("6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf")] # UUID of BenchmarkTools
ERROR: KeyError: key UUID("6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf") not found

julia> Pkg.dependencies()
Dict{Base.UUID, Pkg.API.PackageInfo} with 1 entry:
  UUID("...") => PackageInfo("Example", ...)
```
Just remembered that in recent Julias there is also `pkgversion` from Base
```
julia> using BenchmarkTools

julia> pkgversion(BenchmarkTools)
v"1.3.2"
```